dev/io.openliberty.microprofile.telemetry.2.1.internal_fat_tck/publish/servers/Telemetry21LogsConfigTCKServer/server.xml

```diff
@@ -0,0 +1,47 @@
+<!--Copyright (c) 2025 IBM Corporation and others. All rights reserved.
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License 2.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<server>
+    <featureManager>
+        <feature>componenttest-2.0</feature>
+        <feature>localConnector-1.0</feature>
+        <feature>cdi-4.0</feature>
+        <feature>restfulWS-3.1</feature>
+        <feature>jsonb-3.0</feature>
+        <feature>mpConfig-3.1</feature>
+        <feature>mpRestClient-4.0</feature>
+        <feature>arquillian-support-jakarta-2.1</feature>
+        <feature>ssl-1.0</feature>
+        <!-- Servlet required since we cannot use rest protocol because rest requests create spans which results in more spans than the tests expect -->
+        <!-- See https://github.com/eclipse/microprofile-telemetry/issues/72 -->
+        <feature>servlet-6.0</feature>
+        <feature>mpTelemetry-2.1</feature>
+        <feature>concurrent-3.0</feature>
```

Comment:
Is concurrent-3.0 needed here and in the other server.xml?  Jakarta Concurrency isn't part of Core Profile so I wouldn't think it would be needed for the tests unless there are specific concurrency tests above what is required for Core Profile.